src/abi/rinkeby/StakingHelper.json

```diff
@@ -0,0 +1,98 @@
+{
+  "address": "0x179C45D4c6F8370c68A53aF068b5Fa20e3fE2Af4",
+  "abi": [
```

Comment:
@0xtaichi For every contract, we should have one ABI only instead of having the same ABI under each network folder right? Then the contract addresses should live in the constants files?